ostd/src/arch/loongarch/trap/trap.S

```diff
@@ -0,0 +1,176 @@
+/* SPDX-License-Identifier: MPL-2.0 */
+
+
+# Constants / Macros defined in Rust code:
+#   XLENB
+#   LOAD_SP
+#   STORE_SP
+
+
```

Comment:
There is no reason to have two blank lines here. The same goes for the beginning of the file.